Packs/DNSDB/Integrations/DNSDB_v2/DNSDB_v2.py

```diff
@@ -438,7 +537,7 @@ def lookup_to_markdown(results: List[Dict], want_bailiwick=True) -> str:
         row['FromZoneFile'] = str("zone_time_first" in result)
         out.append(row)
 
-    return tableToMarkdown('Farsight DNSDB Lookup', out, headers=headers)
+    return tableToMarkdown(title, out, headers=headers)
```

Comment:
What happens if a search (either flex or rdata fetch) returns empty? Should be mentioned specifically that search returned empty.

Reply:
tableToMarkdown handles this nicely

![image](https://user-images.githubusercontent.com/6068107/97172647-10c32d00-176e-11eb-9638-1ae3c69657ad.png)